src/components/MenuBar/MenuBar.css

```
@@ -0,0 +1,29 @@
+.menu-bar {
+  border-radius: 4px;
+  background-color: #ec2578;
+  width: 335px;
+  height: 60px;
+  padding: 10px 20px;
+
+  display: flex;
+  gap: 45px;
+}
+
+.home-icon {
+  width: 40px;
+  height: 40px;
+  padding: 6px;
+  border-radius: 4px;
+  background-color: #fff;
+}
+
+.profile-icon,
+.cart-icon,
+.chat-icon {

```

Já que todos tem a mesmas regras CSS tu pode chamar tudo com o mesmo nome `.icon`, e se caso tenha alguma variação, tu cria o nome mais especifico :D 

```suggestion
.icon {
```